k8s-daemonset/k8s/hello-world-latency.yml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: hello
+spec:
+  replicas: 1
```

Comment:
I explicitly set replicas to 1 for this example because I want to be able to call `setLatency` to modify the latency on the hello and world services.  With multiple replicas it would be trickier to make sure the `setLatency` call gets to all of the replicas.